s3/pkg/meta/db/drivers/tidb/bucket.go

```diff
@@ -483,6 +483,12 @@ func (t *TidbClient) DeleteBucket(ctx context.Context, bucket *Bucket) error {
 	_, err := t.Client.Exec(sqltext, bucket.Name)
 	if err != nil {
 		return handleDBError(err)
+	} else {
+		sqltext := "delete from bucket_versionopts where bucketname=?;"
+		_, err := t.Client.Exec(sqltext, bucket.Name)
+		if err != nil {
```

Comment:
If bucket has no versioning set, If there is no entry in the bucket_versionopts for the bucket then this will throw error. This is not correct

Reply:
For all buckets, whether versioned or not, bucket_versionopts will have an entry with status enabled/disabled , so this won't be a problem